airbyte-integrations/connectors/source-pinterest/source_pinterest/spec.json

```diff
@@ -36,6 +36,16 @@
         "title": "Start Date",
         "description": "A date in the format YYYY-MM-DD. If you have not set a date, it would be defaulted to 2020-07-28.",
         "examples": ["2020-07-28"]
+      },
+      "status": {
+        "title": "Status",
+        "description": "Statuses based off of your Pinterest entity. If you do not have a status set, it will be ignored completely.",
```

Comment:
This is going to impact only a subset of streams? Let this clear here.

Reply:
Thanks for catching that, I adjusted the description in the latest commit.